src/metabase/analytics/prometheus.clj

```diff
@@ -264,6 +283,13 @@
      :labels labels
      :value  (initial-value metric labels)}))
 
+(defn- qualified-vals
+  [m]
+  (update-vals m (fn [v] (cond
+                           (map? v) (qualified-vals v)
+                           (keyword? v) (u/qualified-name v)
+                           :else v))))
```

Comment:
My first thought was "why not use `clojure.walk`", but the problem is it'll process the keys. This makes me think that it would be nice to extract a helper called something like `u/walk-values` that just skips transforming keys. Way out of scope of course, but does that sound useful to you?

Reply:
what do you think the api should be here?

I think my use case is pretty specific in that i only want to update when value is a keyword, so If i want this to be re-usable then the api will be `(walk-values m transform-value-if-keyword?)`, which i don't think is useful enough for general audience.